repo-enable.ks.in

```diff
@@ -0,0 +1,64 @@
+#test name: repo-enable
+
+## Test for enabling repositories using repo --name.
+
+%ksappend common/common_no_payload.ks
+%ksappend repos/default.ks
+
+# Enable the repository in the installation environment.
+repo --name=test01
+
+%packages
+a-addon
+%end
+
+%pre
+
+# Create additional repositories in the installation environment.
```

Comment:
Is that a technique that tests the functionality meaningfully? I can imagine using another disk would be the same, except for the hassle of setting it up for the test. But I am not sure what could be left out, just asking. Maybe there could be more about this in the comment(s)?